Field_Simulation/Simulation/WindowsFormsApp1/Form1.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.Threading;
+
+namespace WindowsFormsApp1
+{
+    public partial class Form1 : Form
+    {
+        Mutex cleanM = new Mutex();
+        List<int[]> toClean = new List<int[]>();
+        int vertical = 0;
+        const int CIRCLE_TIME = 1 * 60 ;
+        const int FIELD_SIZE = 50;
+        Bitmap bitmap;
+        DateTime time;
+        public Form1()
+        {
+            InitializeComponent();
+        }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            time = DateTime.Now;
+            this.bitmap = new Bitmap(pictureBox1.Width, pictureBox1.Height);
+            pictureBox1.Image = this.bitmap;
+            for(int i=0;i<this.bitmap.Width;i++)
+            {
+                for(int j=0;j<this.bitmap.Height;j++)
+                {
+                    this.bitmap.SetPixel(i, j, Color.White);
+                }
+            }
+            Thread t = new Thread(() =>
+            {
+                try
+                {
+                    while (true)
+                    {
+                        //removes older vectors
+                        //Clean();
+                        double angle = 360 * get_ratio();
+                        angle = degrees_to_radians(angle);
+                        int[] arr = circular_field(angle);
```

Comment:
Naming convention for the function `circular_field`. Should be `CircularField`.

This is C# here, we use Pascal Case.